api/src/main/java/org/apache/cloudstack/api/command/admin/network/CreateNetworkOfferingCmd.java

```diff
@@ -294,6 +302,10 @@ public String getNsxMode() {
         return nsxMode;
     }
 
+    public Boolean getNsxSupportsLbService() {
```

Comment:
I think we can simplify the usages of this method by making it return a boolen instead of Boolean